### ISSUE TITLE
Do accessibility audit and/or scan of 5.0 release

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#279]
Using the wave tool and level access' tools perform an accessibility review of everything to date in version 5.

**Comment by sherakama** [comment]
* Use level Access and set up regular scans of http://decanter.stanford.edu/
* Scans should be of both the whole site and of each specific component.  
* Parse the results of the scans and create issues in the github issue queue.

**Comment by sherakama** [comment]
Closing in favor of the granular tickets.
